src/MessagePack.UnityClient/Assets/Scripts/MessagePack/MessagePackSerializerOptions.cs

```diff
@@ -28,7 +28,11 @@ public class MessagePackSerializerOptions
             "System.Management.IWbemClassObjectFreeThreaded",
         };
 
-#if !DYNAMICCODEDUMPER
+#if DYNAMICCODEDUMPER
+        static readonly MessagePackSerializerOptions _standard = new MessagePackSerializerOptions(Resolvers.BuiltinResolver.Instance);
```

Comment:
FYI: This code fails on CI

Reply:
Main goal to add code lines in this place is to have Standard  MessagePackSerializerOptions when projects are build with DYNAMICCODEDUMPER symbol.

I am using this Standard static property for example to init MessagePack like this:

```
public static class MessagePackInitiator
{
	static Boolean _resolverWasRegistered = false;         

	public static void Run(MessagePackHubProtocolOptions options)
	{
		if (!_resolverWasRegistered)
		{
			StaticCompositeResolver.Instance.Register(
				GeneratedResolver.Instance,
				BuiltinResolver.Instance);

			_resolverWasRegistered = true;
		}           

		options.SerializerOptions = MessagePackSerializerOptions.Standard
			.WithResolver(StaticCompositeResolver.Instance)
			.WithSecurity(MessagePackSecurity.UntrustedData);
	}
	
}
```

But it appeared that it is not so easy to add this property here, because DynamicCodeDumper project has not some files.
Do not want to add missed files. Maybe MessagePack authours will do it.
Instead just deleted my changes.
And instead will init MessagePack like this in my code:


```
public static class MessagePackInitiator
{
	static Boolean _resolverWasRegistered = false;         

	public static void Run(MessagePackHubProtocolOptions options)
	{
		if (!_resolverWasRegistered)
		{
			StaticCompositeResolver.Instance.Register(
				GeneratedResolver.Instance,
				BuiltinResolver.Instance);

			_resolverWasRegistered = true;
		}           

		options.SerializerOptions =  new MessagePackSerializerOptions(Resolvers.BuiltinResolver.Instance)
			.WithResolver(StaticCompositeResolver.Instance)
			.WithSecurity(MessagePackSecurity.UntrustedData);
	}
	
}
```